### PR TITLE
Fix to resource spawn rates

### PR DESCRIPTION
**Pull request by GuybrushGit**
Fixes the respawn timers for all herbs and ores.

Added 2 custom SQLs to double the rate or reset it back to default.

### CODE REVIEW

File: sql/migrations/20171019212730_world.sql [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20171019212730');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20171019212730');
+-- Add your query below.
+
+-- low level ores
+UPDATE `gameobject` SET `spawntimesecs`=900 WHERE `id` IN (1610,1667,1731,1732,1733,2054,2055,3763,3764,73940,103711,103713,1731,1732,1733,1734) and map IN (0,1);
```
Comment (thepparker) [inline sql/migrations/20171019212730_world.sql]: gold (1734) is in the query for low level and mid-level ores

Comment (GuybrushGit) [reply sql/migrations/20171019212730_world.sql]: fixed

**Comment by rickharrison** [comment]
Which of these migrations is going to be applied to Elysium PvP?

**Comment by Eegor74** [comment]
On elysium black lotus spawn times were 60 min if population <2500 and if population > 2500 then 20 min then you use the formula: spawn time x blizzlike / current population. In response to https://github.com/LightsHope/server/issues/107 . their was nothing wrong with how the spawn time was calculated 

(1200*2500)/7500=400 sec which 6 min 40 sec

This change will increase spawns times by 50% and prices will skyrocket, don't fix what isnt broke.

**Pull request merged**